src/olympia/blocklist/views.py

```diff
@@ -132,6 +156,46 @@ def between(ver, min, max):
 
 
 @non_atomic_requests
+def blocklist_json(request):
+    key = 'blocklist:json'
+    cache.add('blocklist:keyversion', 1)
+    version = cache.get('blocklist:keyversion')
```

Comment:
Apparently cache.add doesn't return the value of the key: https://docs.djangoproject.com/en/1.9/topics/cache/#basic-usage